plugins/Rockets/RocketsPlugin.cpp

```diff
@@ -284,8 +290,9 @@ class RocketsPlugin::Impl : public ActionInterface
                          const PropertyMap& input, const PropertyMap& output,
                          const std::function<PropertyMap(PropertyMap)>& action)
     {
-        _bindEndpoint(desc.methodName, [ name = desc.methodName,
-                                         action ](const auto& request) {
+        _bindEndpoint(desc.methodName, [ name = desc.methodName, action,
+                                         this ](const auto& request) {
+            ScopedCurrentClient scope(this->_currentClientID, request.clientID);
```

Comment:
not needed imo, as `_bindEndpoint` does it.

Reply:
Indeed